src/components/FloatingHelper/HelperContent/HelperContent.tsx

```diff
@@ -3,36 +3,44 @@ import { string } from "prop-types";
 import style from './HelperContent.st.css';
 import {Text} from '../../../components/Text';
 import {DataHooks} from './DataHooks';
-
+import { Button } from "../../Button";
```

Comment:
can you please check how come the lint rule didn't work here. we would like to have a lint rule for double quotes. I can also take a look if you are not in the mood :)